numpy/core/src/umath/funcs.inc.src

```diff
@@ -226,6 +226,17 @@ npy_ObjectLCM(PyObject *i1, PyObject *i2)
     return PyNumber_Absolute(tmp);
 }
 
+
+static PyObject *
+npy_ObjectClip(PyObject *arr, PyObject *min, PyObject *max) {
+    PyObject *o = npy_ObjectMax(arr, min);
+    if (o == NULL) {
+        return NULL;
+    }
+    Py_SETREF(o, npy_ObjectMin(arr, max));
```

Comment:
This should be `npy_ObjectMin(o, max)`! Also means there is no test yet that catches this...

Reply:
Oops!

Reply:
Ok, patched with regression test now, so marking as resolved.